web/src/components/InputSection.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { HTMLInputTypeAttribute, ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import TextInput from './TextInput'
+
+const MULTI_LINE_NUMBER = 7
```

Comment:
i would use an optional parameter with a default value instead of setting it here fixed to 7 for every multiline field